card_generator/api/v1/cards/views.py

```diff
@@ -34,6 +35,14 @@ class CardViewSet(ModelViewSet):
     authentication_classes = (TokenAuthentication,)
     http_method_names = ("get", "post", "put", "delete")
 
+    def get_object(self):
+        card_data = cache.get("card_data")
```

Comment:
How does it work? Do you cache all the card objects under the name `card_data`? Isn't it will always return the same card?